cmd/sriov/main.go

```diff
@@ -169,15 +164,20 @@ func cmdDel(args *skel.CmdArgs) error {
 		}
 	}()
 
-	sm := sriov.NewSriovManager()
-
 	if netConf.IPAM.Type != "" {
 		err = ipam.ExecDel(netConf.IPAM.Type, args.StdinData)
 		if err != nil {
 			return err
 		}
 	}
 
+	// https://github.com/kubernetes/kubernetes/pull/35240
+	if args.Netns == "" {
```

Comment:
At this time the interface would just be moved into default net namespace (in case of net device) due to container exit, but we still need to rename the interface back to original name.

Reply:
That issue, if exists, is then orthogonal to what this PR is trying to fix, right? If `rename of an interface` is required, then it should be done in a separate PR.